Source/Particles/PhysicalParticleContainer.H

```diff
@@ -221,6 +221,23 @@ public:
         amrex::Gpu::HostVector<amrex::ParticleReal>& particle_uz,
         amrex::Gpu::HostVector<amrex::ParticleReal>& particle_w);
 
+    /**
+     * \brief Default initialize runtime attributes in a tile.
```

Comment:
Overall, this looks good. For clarity, perhaps add a comment here that this routine is only setting attributes that have a potentially nonzero default value and that were not set elsewhere.

Reply:
Thanks, I've updated this comment to state that this does not initialize attributes that were set elsewhere. (actually I think that this routine does zero initialize attributes that should be zero initialized, like `prev_x`, so I haven't added a comment on that part)